src/CalcManager/CEngine/scicomm.cpp

```diff
@@ -30,7 +30,7 @@ using namespace CalcEngine;
 //
 //  returns a virtual number for precedence for the operator. We expect binary operator only, otherwise the lowest number
 // 0 is returned. Higher the number, higher the precedence of the operator.
-INT NPrecedenceOfOp(int nopCode)
+static INT NPrecedenceOfOp(int nopCode)
```

Comment:
I think the better fix here is to move these functions to an anonymous namespace.  Same for the other instances below.